entrypoint/versionbump/canal_test.go

```diff
@@ -32,22 +32,22 @@ var _ = Describe("VersionTemplate Upgrade:", func() {
 	})
 
 	It("Verifies bump version on rke2 for canal with calico and flannel versions", func() {
-		template.VersionTemplate(template.VersionTestTemplate{
-			TestCombination: &template.RunCmd{
-				Run: []template.TestMap{
+		Template(TestTemplate{
```

Comment:
Can we keep as is `VersionTemplate` or rename to `VersionTestTemplate` as this template only refers to versions?
If we want to use TestTemplate, then can we do `TestTemplate.Version{}`?

Reply:
Hey the idea here is to start decoupling this from testing only versions
this would be a first step for this future request https://github.com/rancher/qa-tasks/issues/980

The pkg name already says that is a version bump + the name itself of the test